docs/resources/iotda_batchtask_file.md

```diff
@@ -0,0 +1,64 @@
+---
+subcategory: "IoT Device Access (IoTDA)"
+---
+
+# huaweicloud_iotda_batchtask_file
+
+Manages an IoTDA batch task file within HuaweiCloud.
+
+## Example Usage
+
+```hcl
+resource "huaweicloud_iotda_batchtask_file" "test" {
+  content = "./BatchCreateDevices_Template.xlsx"
+}
+```
+
+## Argument Reference
+
+The following arguments are supported:
+
+* `region` - (Optional, String, ForceNew) Specifies the region in which to create the IoTDA batch task file resource.
+  If omitted, the provider-level region will be used. Changing this parameter will create a new resource.
+
+* `content` - (Required, String, ForceNew) Specifies the path to the batch task file to be uploaded.
+  Currently, only the **xlsx/xls** file format is supported, and the maximum number of lines in the file is `30000`.
+  The file name cannot be duplicated. Changing this parameter will create a new resource.
+  Please following [reference](https://support.huaweicloud.com/usermanual-iothub/iot_01_0032.html),
```

Comment:
It is recommended to use the English version of the document link.
https://support.huaweicloud.com/intl/en-us/usermanual-iothub/iot_01_0032.html

Reply:
fixed.